src/betterproto/grpc/grpclib_server.py

```diff
@@ -0,0 +1,29 @@
+from abc import ABC
+
+import grpclib
+import grpclib.server
+
+
+class ServiceImplementation(ABC):
+    """
+    Base class for async gRPC servers.
+    """
+
+    __service_name__: str
+
+    def __rpc_methods__(self):
+        pass
+
+    def __mapping__(self):
```

Comment:
are return type annotations possible for this method and the above?

Reply:
I removed `__rpc_methods__` in my latest commit. `__mapping__` now has typing hints.